dev/com.ibm.ws.security.openidconnect.clients.common/src/com/ibm/ws/security/openidconnect/clients/common/AuthorizationCodeHandler.java

```diff
@@ -59,6 +60,7 @@ public AuthorizationCodeHandler(HttpServletRequest request, HttpServletResponse
         this.response = response;
         this.clientConfig = clientConfig;
         clientId = clientConfig.getClientId();
+        configId = clientConfig.getId();
```

Comment:
why are we using `configId` instead of `clientId` in these messages now? do the NLS messages have to be updated too since they use language like `The OpenID Connect client [{1}] is...`? i feel like this may possibly be confusing in the future.

Reply:
I noticed this when I was looking at the new error message we were emitting. We can have multiple `<openidConnectClient>` elements configured that all use the same client ID, i.e. the same OAuth client that's registered with the OP. However each `<openidConnectClient>` element has its own unique ID that's distinct from whatever client ID is being used by that configuration. We want the unique ID to be in these messages so that administrators know which `<openidConnectClient>` configuration is actually causing the problem.

We've got tests, for example, that have all of these configurations in the same server.xml:
```XML
    <oidcLogin clientId="client01" id="oidcLogin1" authFilterRef="authFilter1" ... />
    <oidcLogin clientId="client01" id="oidcLogin_badClientSecret" authFilterRef="authFilter_badClientSecret" ... />
    <oidcLogin clientId="client01" id="oidcLogin_blankClientSecret" authFilterRef="authFilter_blankClientSecret" ... />
    <oidcLogin clientId="client01" id="oidcLogin_emptyClientSecret" authFilterRef="authFilter_emptyClientSecret" ... />
    <oidcLogin clientId="client01" id="oidcLogin_XOR_Secret" authFilterRef="authFilter_xorSecret" ... />
   ...
```

They all use the same client ID, so before my change the error message would end up being `The OpenID Connect client [client01] is unable to contact the OpenID Connect provider...`, which doesn't point to which configuration actually had the problem. Now the message will be something like `The OpenID Connect client [oidcLogin_badClientSecret] is unable to contact the OpenID Connect provider...`, which points to the actual config being used.